docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -0,0 +1,466 @@
+---
+description: Ibexa DXP v4.6 brings improvements to Commerce, PIM and Personalization offerings, and a number of changes in CDP and Ibexa Connect.
+---
+
+<!-- vale VariablesVersion = NO -->
+
+
+# Ibexa DXP v4.6
+
+**Version number**: v4.6
+
+**Release date**: July 20, 2023
```

Comment:
As mentioned on Slack, this must be fixed.

Reply:
Once we know the date, we will change it